components/style/animation.rs

```diff
@@ -509,9 +537,10 @@ impl Animation {
         match self.state {
             AnimationState::Running => {},
             AnimationState::Finished => return true,
-            AnimationState::Pending | AnimationState::Canceled | AnimationState::Paused(_) => {
-                return false
+            AnimationState::Paused(progress) => {
+                return self.on_last_iteration() && progress > self.current_iteration_length();
```

Comment:
Should it be `>=`?

Reply:
I've corrected this, which I think was a weird side-effect of the fact that we used to iterate past the last expected iteration when animations finished. I've corrected that and updated to code to reflect it. I think things should be less confusing now. This has also led me to make some improvements to how we calculate `elapsedTime` for animation events. It should be correct now for fractional iteration counts.